rpmconf/Install/zmsetup.pl

```diff
@@ -7841,6 +7842,21 @@ sub addJDK17Options {
 	}
 }
 
+sub removeProductHelpLink {
+  if (isInstalled("zimbra-store")) {
+    progress( "modifying the help link config globally...");
+    my $rc = main::runAsZimbra("$ZMPROV mcf -zimbraWebClientSupportedHelps productHelp");
+    progress(($rc == 0) ? "done.\n" : "failed.\n");
+    main::runAsZimbraWithOutput("$ZMPROV gad");
+    my $domains = $?;
```

Comment:
@umagmrit @dasiyogesh @silentsakky can anyone help me with this code ? This line returns `0` `main::runAsZimbraWithOutput("$ZMPROV gad");`. I basically want it to return the output of "$ZMPROV gad" command. I guess its not returning the output because of the final nested command generated by the `runAsZimbraWithOutput` method which looks like this - `system("su - zimbra -c \"zmprov gad\"")`
Hence it does not catches the output of the "zmprov gad" command.  Let me know if anyone knows a way to get the result of "zmprov gad" command.